ecc/blocks/form-handler/form-handler.js

```diff
@@ -303,7 +303,11 @@ function showSaveSuccessMessage(props) {
 }
 
 async function saveEvent(props, options = { toPublish: false }) {
-  await gatherValues(props);
+  try {
+    await gatherValues(props);
+  } catch (e) {
+    return { message: e.message };
```

Comment:
Do we want to include errors here too?

Reply:
this is an error which is thrown by UI element and not coming from service. service errors or messages are handled in respData body